Setup/Patch/Data/RelationDataId.php

```diff
@@ -98,7 +98,7 @@ public function apply()
 
             $this->attributeResource->save($attribute);
         } catch (\Exception $e) {
-            $this->logger->error($e->getMessage());
+            $this->logger->error($e->getMessage(), $this->logger->buildReportData($e));
```

Comment:
why do we even have the catch here ?

Reply:
My point was more if should not just let it fail, now when you upgrade and it's broken people won't figure out untill the order fails for example